public/app/features/dashboard/components/PanelEditor/PanelOptionsTab.tsx

```diff
@@ -48,7 +48,7 @@ export const PanelOptionsTab: FC<Props> = ({
   };
   // Fist common panel settings Title, description
   elements.push(
-    <OptionsGroup title="Panel settings" key="Panel settings">
+    <OptionsGroupWithPersistence title="Panel settings" id="Panel settings" key="Panel settings">
```

Comment:
Wonder if the persistence shouldn't be controlled via prop. But that's a nit

Reply:
If you had the prop then you could have it set to true by default and not care about replacing OptionsGroup usage

Reply:
Yea, that was what I was thinking as we’ll

Reply:
I think this is a lot clearer, and I found the OptionsGroup API already so cluttered so I went this path instead.

Reply:
For this to be just a built in behavior in the normal OptionsGroup component with option to disable via prop

Reply:
I would even go further and refacto OptionsGroup to use something like CollapsibleSection that would hide this behaviour. But maybe that's too much...? :D

Reply:
Ok, I'll pass it as a prop then So 2 new props on OptionsGroup then `id` and `persistMe`?

Reply:
ThumbsUp™